app/config/config.yml

```diff
@@ -91,14 +91,6 @@ twig:
     modify_all_shops_prefix: !php/const PrestaShopBundle\Form\Extension\ModifyAllShopsExtension::MODIFY_ALL_SHOPS_PREFIX
     disabling_switch_prefix: !php/const PrestaShopBundle\Form\Extension\DisablingSwitchExtension::FIELD_PREFIX
 
-# Swiftmailer Configuration
-swiftmailer:
```

Comment:
Just one question @lartist these `mailer_*` parameters are not used anymore at all? Were they even useful since it seems the configuration was stored in the DB and set on runtime anyway 🤔
Or should be backport them into the Mailer config in some way?

If they're completely useless maybe they should be cleaned from the default parameters as well? (not blocking for this PR though but nice to have)

Reply:
Ok I guess it's enough for this PR we can clean them in another PR